### PR TITLE
Fixed issue with oc process command and Added KIP_IMAGE parameter to configmap.yaml

### PR DESCRIPTION
**Pull request by Sachin-Itagi**
Related Issue: https://issues.redhat.com/browse/CRW-5768

The commit modifies the apiVersion to fix the issue with the oc process command.
Also Added the KIP_IMAGE parameter that is missing in the configmap.yaml file.

### CODE REVIEW

File: deploy/openshift/configmap.yaml [hunk 1]
```diff
@@ -48,3 +49,5 @@ parameters:
   value: ""
 - name: AFFINITY
   value: "{}"
+- name: KIP_IMAGE
+  value: quay.io/eclipse/kubernetes-image-puller:next
```
Comment (ibuziuk) [inline deploy/openshift/configmap.yaml:53]: no new line

**Review by ibuziuk — APPROVED** [review]
@dkwon17 could you please take a look

**Review by dkwon17 — APPROVED** [review]

**Comment by dkwon17** [comment]
Thank you @Sachin-Itagi I believe we can merge after a new line is added at the end of `deploy/openshift/configmap.yaml`

**Comment by Sachin-Itagi** [comment]
@dkwon17 @ibuziuk Added a new line to configmap.yaml. Can you please check? Thanks.

File: deploy/openshift/configmap.yaml (hunk 1 above)
Comment (Sachin-Itagi) [reply deploy/openshift/configmap.yaml]: Added a new line

File: deploy/openshift/configmap.yaml [hunk 2]
```diff
@@ -48,3 +49,6 @@ parameters:
...
+
```
Comment (ibuziuk) [inline deploy/openshift/configmap.yaml:54]: @Sachin-Itagi this extra line should be removed for template sanity e.g. there should be no red marks on the PR files

File: deploy/openshift/configmap.yaml (hunk 1 above)
Comment (ibuziuk) [inline deploy/openshift/configmap.yaml:53]:
```suggestion
  value: quay.io/eclipse/kubernetes-image-puller:next

```

**Pull request merged**